README.md

```diff
@@ -38,13 +38,13 @@ If you have a running [backend](https://github.com/adfinis-sygroup/timed-backend
   or
 - `yarn start`
 
-If you are using docker-compose you can start the frontend and backend by running
+If you are using docker-compose you can start the frontend and backend by running the following in the backend project
 
 ```
 docker-compose up -d --build
 ```
 
-The frontend will be under [http://localhost:4200](http://localhost:4200) and the backend [http://localhost:8000](http://localhost:8000)
+The frontend will be under [http://timed.local](http://timed.local) and the backend [http://timed.local/admin/](http://timed.local/admin/)
```

Comment:
Is this really necessary?

Reply:
Actually no, I could also just give a link to the backend project there. Can do it if you wish so.

Reply:
As discussed: Ok :-)